widget.go

```diff
@@ -366,6 +433,10 @@ var widgets = map[string]widgetInfo{
 			return []*widget.FormItem{
 				widget.NewFormItem("Value", val)}
 		},
+		gostring: func(obj fyne.CanvasObject) string {
+			slider := obj.(*widget.Slider)
+			return fmt.Sprintf("widget.Slider(Min:0, Max:100, Value:%f)", slider.Value)
```

Comment:
This doesn't look like valid Go code (Either move to NewSlider, or change ( to { and add &).
I suspect we should add a test to catch things like this.